src/db/DB.js

```diff
@@ -2,13 +2,15 @@ import Dexie from "dexie";
 
 export const db = new Dexie("fit_car");
 
-db.version(5).stores({
+db.version(6).stores({
   mileage: "++id, [carId+timeUtc], carId, mileage, timeUtc",
   cars: "++id, name, make, style, maxTank, isActive",
   fuelFillUps:
     "++id, [carId+timeUtc], carId, price, fuelType, previousTank, currentTank, timeUtc",
   events: "++id, carId, type, eventId, timeUtc",
-  oilChecks: "++id, [carId+timeUtc], carId, timeUtc, percentage, opacity, note",
+  oilChecks: "++id, [carId+timeUtc], carId, timeUtc, percentage, opacity, action, note",
+  tireChecks:
+    "++id, [carId+timeUtc], topLeft, topRight, bottomLeft, bottomRight, carId, pressure, timeUtc",
```

Comment:
Take note that `topLeft`, `topRight`, `bottomLeft` and `bottomRight` are the previous tire values and the current pressure at this point is time is `pressure`.

It is assumed that the pressure is pumped out to all the tires and the recordings really are just what the tires are at, at the current time.